shared/actions/teams.js

```diff
@@ -932,36 +934,22 @@ function _updateChannelname(action: TeamsGen.UpdateChannelNamePayload, state: Ty
     identifyBehavior: RPCTypes.tlfKeysTLFIdentifyBehavior.chatGui,
   }
 
-  return Saga.all([
+  return Saga.sequentially([
     Saga.put(createIncrementWaiting(waitingKey)),
     Saga.call(RPCChatTypes.localPostMetadataRpcPromise, param),
-    Saga.identity(
-      Saga.all([
-        Saga.put(createDecrementWaiting(waitingKey)),
-        Saga.put(TeamsGen.createGetChannels({teamname})),
-      ])
-    ),
+    Saga.put(createDecrementWaiting(waitingKey)),
   ])
 }
 
 function _deleteChannelConfirmed(action: TeamsGen.DeleteChannelConfirmedPayload, state: TypedState) {
   const {conversationIDKey} = action.payload
-  const channelName = Constants.getChannelNameFromConvID(state, conversationIDKey)
-  const teamname = Constants.getTeamNameFromConvID(state, conversationIDKey) || ''
-
-  if (!channelName) {
-    return
-  }
-  const param = {
+  // channelName is only needed for confirmation, so since we handle
```

Comment:
hmm, we may need to do something here to update the set of convIDs for this team